hack/arktos-worker-up.sh

```diff
@@ -68,6 +68,7 @@ sudo ./_output/local/bin/linux/amd64/hyperkube kubelet \
 --container-runtime-endpoint="containerRuntime,container,/run/containerd/containerd.sock;vmRuntime,vm,/run/virtlet.sock" \
 --runtime-request-timeout=2m \
 --port=10250 \
+--resolv-conf=/run/systemd/resolve/resolv.conf \
```

Comment:
since RESOLV_CONF is already the env var for resolv file path, can we use that instead of hard coding at this place?

Reply:
The script hack/arktos-worker-up.sh does not use the two environment setting files - hack/lib/common-var-init.sh including  RESOLV_CONF and hack/lib/common.sh which are used by the script hack/arktos-up.sh. So --resolv-conf=/run/systemd/resolve/resolv.conf is added here.